casper/src/test/resources/BlockDataContractTest.rho

```diff
@@ -0,0 +1,32 @@
+new
+  rl(`rho:registry:lookup`), RhoSpecCh,
+  getBlockData(`rho:block:data`),
```

Comment:
I think this should be `rho:rchain:blockData` or `rho:rchain:pos:blockData`

Reply:
since we have rho:deploy:params it makes sense to have here rho:block:data. If we change one then we should change both

Reply:
you're goddamn right 'bout that. Please change `rho:deploy:params` as well then. I'm fine if the latter is changed in a different PR.

Reply:
I'll change both in a separate PR

Reply:
I ticketed this work: https://rchain.atlassian.net/browse/RCHAIN-3554